test/functional/varint.js

```diff
@@ -80,42 +97,128 @@ describe('Varint tables', function() {
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { lt: 2 }
                 }
-            });
+            }
         })
         .then(function(result) {
             deepEqual(result.body.items.length, 1);
+            deepEqual(result.body.items[0].key, 'testing');
+            deepEqual(result.body.items[0].rev, 1);
+        });
+    });
+
+    it('retrieves using gt predicate', function() {
+        return router.request({
+            uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
+            method: 'get',
+            body: {
+                table: 'varintTable',
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { gt: 1 }
+                }
+            }
         })
-        .then(function () {
-            return router.request({
-                uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
-                method: 'get',
-                body: {
-                    table: 'varintTable',
-                    limit: 3,
-                    attributes: {
-                        key: 'testing',
-                        rev: { ge: 1 }
-                    }
+        .then(function(result) {
+            deepEqual(result.body.items.length, 1);
+            deepEqual(result.body.items[0].key, 'testing');
+            deepEqual(result.body.items[0].rev, 5);
+        });
+    });
+
+    it('retrieves using le predicate', function() {
+        return router.request({
+            uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
+            method: 'get',
+            body: {
+                table: 'varintTable',
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { le: 5 }
                 }
-            });
+            }
+        })
+        .then(function(result) {
+            deepEqual(result.body.items.length, 2);
+        });
+    });
+
+    it('retrieves using ge predicate', function() {
+        return router.request({
+            uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
+            method: 'get',
+            body: {
+                table: 'varintTable',
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { ge: 1 }
+                }
+            }
         })
         .then(function(result) {
             deepEqual(result.body.items.length, 2);
         });
     });
+
+    it('retrieves using multiple predicates', function() {
+        return router.request({
+            uri: '/restbase.cassandra.test.local/sys/table/varintTable/',
+            method: 'get',
+            body: {
+                table: 'varintTable',
+                limit: 3,
+                attributes: {
+                    key: 'testing',
+                    rev: { eq: 1 },
```

Comment:
Yes, exactly, because that order is not defined (in Cassandra). Forgot to mention in the comment that that test should fail :P